betterproto/tests/grpc/test_grpclib_client.py

```diff
@@ -1,14 +1,18 @@
 import asyncio
+import sys
+
+import grpclib
+import grpclib.metadata
```

Comment:
redundant import?

Reply:
Intentional. Without it, inspection will complain. Think `__all__` is defined in `grpclib/__init__.py`.

Reply:
right, though I didn't see anything using grpclib.metadata specifically

Reply:
It is used by existing test cases. See L59 for example below.

https://github.com/danielgtaylor/python-betterproto/blob/0cf8aaab6bddbdebcb0602b4ca36428d4cf70a5a/betterproto/tests/grpc/test_grpclib_client.py#L56-L124